.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+_Please use this to only for bug reports. For questions or when you need help, you can use the [GitHub Discussions](https://github.com/kroxylicious/design/discussions)
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Configure kroxylicious like this ...
+2. Run command '....'
+3. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**YAML files and logs**
+
+Attach or copy paste your kroxylicious configuration YAML.
```

Comment:
Should we recommend removing cluster addresses etc?

Reply:
Added a bit about redacting sensitive info